src/main/java/com/example/Employee.java

```diff
@@ -0,0 +1,63 @@
+package com.example;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class Employee {
+    private String employeeId;
+    private String name;
+    private String position;
+    private double salary;
+
+    public Employee(String employeeId, String name, String position, double salary) {
+        this.employeeId = employeeId;
+        this.name = name;
+        this.position = position;
+        this.salary = salary;
+    }
+
+    public String getEmployeeId() {
+        return employeeId;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getPosition() {
+        return position;
+    }
+
+    public double getSalary() {
+        return salary;
+    }
+
+    public void promoteEmployee(String newPosition, double newSalary) {
```

Comment:
what if an employee called Rick has a new position that is a demotion (because of his bad java)